mobileversion.css

```diff
@@ -0,0 +1,887 @@
+* {
+  box-sizing: border-box;
+}
+
+.nav-bar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+  margin-left: -20px;
+}
+
+hr {
+  width: 327px;
+  margin-left: -5px;
+}
+
+.log {
+  font-weight: bold;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  text-decoration: none;
+  padding: 2px;
+}
+
+.nav-link {
+  text-decoration: none;
+  transition: 0.7s ease;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  opacity: 1;
+  padding: 0;
+}
+
+.nav-link:hover {
+  color: dodgerblue;
+}
+
+@media screen and (max-width: 768px) {
+  .nav-menu {
+    display: flex;
+    align-items: flex-start;
+    position: fixed;
+    left: 100%;
+    height: 100%;
+    top: -25px;
+    gap: 0;
+    flex-direction: column;
+    background-color: #6070ff;
+    border-radius: 8px;
+    opacity: 0.9;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+    -webkit-backdrop-filter: blur(7px);
+    backdrop-filter: blur(7px);
+  }
+}
+
+/* .nav-menu hr {
+  width: 40%;
+  border: 2px solid black;
+  border-radius: 5px;
+  margin-top: 285px;
+  margin-left: 100px;
+  place-items: center;
+} */
+
+.hamburger {
+  cursor: pointer;
+  margin-left: 350px;
+  position: absolute;
+}
+
+.nav-item {
+  margin-top: 50px;
+  margin-bottom: 20px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: blue;
+}
+
+.hamburger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: white;
+}
+
+.hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: white;
+}
+
+.nav-menu.active {
+  left: 0;
+}
+
+.flex-container {
+  background-image: url("https://user-images.githubusercontent.com/66992901/198218680-01a4bf79-49e0-4ca5-b5a4-363c351466a5.png");
+  display: flex;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  left: 2px;
+  width: 400px;
+  height: 700px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  flex: none;

```

Comment:
- [OPTIONAL ] Kindly make sure your current changes do not alter your previous, the previous first section  should stick to the design as much as possible , you can start by removing the `border-radius:16px`
  -  Your design:
![mainSt](https://user-images.githubusercontent.com/97834160/205039491-2ff064f5-7ed8-4fbd-a11b-da40c2a2e94f.png)

  - Figma design:
![main](https://user-images.githubusercontent.com/97834160/205039577-05bbc223-7dab-4aca-93c0-b03fca5e35ce.png)